_posts/en/newsletters/2018-07-10-newsletter.md

```diff
@@ -0,0 +1,272 @@
+  may be affected.  If you have not yet checked your infrastructure for
+  affected services, it is advised to do so now.  See [newsletter #1][]
+  for more details
+
+[alert released]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-July/016189.html
+[newsletter #1]: https://bitcoinops.org/en/newsletters/2018/06/26/
+
+## Dashboard items
+
+- **Transaction fees remain very low:** as of this writing, fee
+  estimates for confirmation 2 or more blocks in the future remain at
+  roughly the level of the default minimum relay fee in Bitcoin Core.
+  It's a good time to [consolidate inputs][].
+
+[consolidate inputs]: https://en.bitcoin.it/wiki/Techniques_to_reduce_transaction_fees#Consolidation
+
+- **Block production recovery:** following last week's news about
+  flooding in China affecting miner operations, Bitcoin block production
+  seems to have recovered to the expected level of about one block every
+  10 minutes.
+
+## Featured news: Schnorr signature proposed BIP
+
+In a [post][schnorr post] to the bitcoin-dev mailing list, Pieter Wuille
+submitted a [draft specification][schnorr draft] for a Schnorr-based
+signature format.  The goal of the specification is to hopefully get
+everyone in agreement about what Schnorr signatures will look like on
+Bitcoin before work begins on an actual soft fork, so the BIP does not
+propose specific new opcodes, segwit witness flags, soft fork activation
+method, or anything else necessary to make this change part of the
+Bitcoin consensus rules.  However, it is possible to say what this signature
+format will provide if it becomes the form of Schnorr signature adopted by
+Bitcoin.
+
+[schnorr post]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-July/016203.html
+[schnorr draft]: https://github.com/sipa/bips/blob/bip-schnorr/bip-schnorr.mediawiki
+
+1. Full compatibility with existing Bitcoin private keys and public
+   keys, meaning that existing HD wallets that upgrade won't need to
+   generate new recovery seeds.
+
+2. Roughly 10% smaller signatures, providing a slight increase to
+   block chain capacity as Schnorr is adopted.
+
+3. Batch verification of signatures providing a roughly 2x speedup
+   over individual verification for a block full of Schnorr
+   signatures.  This mainly affects nodes initially syncing or
+   catching up after being offline.
+
+4. Full compression and significantly improved privacy for multisig use
+   cases, but with required interaction: an unlimited number of
+   participants can create a single 33-byte public key and 64-byte
+   signature from the combination of their individual public keys and
+   signatures, using secure multisig with the same efficiency of
+   single-sig and increasing their privacy by making multisig look like
+   single-sig.  However, the scheme requires multistep interaction
+   between the wallets participating in the multisig, both for creating
+   the public key and the signature.
```

Comment:
Non-blocking, but it may be worth noting that this functionality creates huge fee incentives for exchanges et al to start batching withdrawals.

Reply:
Ah nevermind - this isn't actually the case (as you mention below) because we're still unable to aggregate a signature across inputs within a txn.